oct/pages/models/checkout_page.py

```diff
@@ -0,0 +1,143 @@
+from pages.base.elements import (
+    RadioButton,
+    Block,
+    InputField,
+    Clickable,
+    CheckBox,
+    Dropdown,
+    Form,
+)
+from oct.pages.base.page import BasePage
+
+
+class CheckoutOptions(Block):
+    contains = {
+        "register_account": {
+            "locator": (
+                "XPATH",
+                '//*[@id="collapse-checkout-option"]/div/div/div[1]/div[1]/label/input',
+            ),
+            "class": RadioButton,
+        },
+        "guest_checkout": {
+            "locator": (
+                "XPATH",
+                '//*[@id="collapse-checkout-option"]/div/div/div[1]/div[2]/label/input',
+            ),
+            "class": RadioButton,
+        },
+        "continue_button": {"locator": ("ID", "button-account"), "class": Clickable},
+    }
+
```

Comment:
You can use RadioButtonGroup class instead of Block
```suggestion
class CheckoutOptions(RadioButtonGroup):
    contains = {
        "register_account": {
            "locator": (
                "XPATH",
                '//*[@id="collapse-checkout-option"]/div/div/div[1]/div[1]/label/input',
            ),
            "class": RadioButton,
        },
        "guest_checkout": {
            "locator": (
                "XPATH",
                '//*[@id="collapse-checkout-option"]/div/div/div[1]/div[2]/label/input',
            ),
            "class": RadioButton,
        },
        "continue_button": {"locator": ("ID", "button-account"), "class": Clickable},
    }
```